src/library/browse/browsefeature.cpp

```diff
@@ -133,7 +133,13 @@ void BrowseFeature::slotAddQuickLink() {
     QVariant vpath = m_pLastRightClickedItem->getData();
     QString spath = vpath.toString();
     QString name = extractNameFromPath(spath);
-    m_pQuickLinkItem->appendChild(name, vpath);
+
+    QModelIndex parent = m_childModel.index(m_pQuickLinkItem->parentRow(), 0);
+    auto pNewChild = new TreeItem(this, name, vpath);
```

Comment:
There is a very low probability that pNewChild item leaks until it has been inserted into the model. The key word 'auto' hides the plain pointer, so I would recommend at least to fix this by making it explicit.

Using a unique_ptr with get() when appending and calling release() after inserting would be fail safe.